WalletWasabi.Tests/UnitTests/WabiSabi/Backend/PostRequests/RegisterInputFailureTests.cs

```diff
@@ -131,6 +131,34 @@ public async Task InputBannedAsync()
 			async () => await arenaClient.RegisterInputAsync(round.Id, outpoint, ownershipProof, CancellationToken.None));
 
 		Assert.Equal(WabiSabiProtocolErrorCode.InputBanned, ex.ErrorCode);
+		Assert.IsAssignableFrom<InputBannedExceptionData>(ex.ExceptionData);
```

Comment:
As @kiminuo mentioned elsewhere, this does not sound really englishy, `BannedInputExceptionData` sounds better.

Same goes for other methods and variables.

Reply:
Feel free to rename them in another PR.

Reply:
it is OK.